spark-connect/client/src/test/scala-spark-master/io/delta/connect/tables/RemoteSparkSession.scala

```diff
@@ -69,17 +69,14 @@ trait RemoteSparkSession extends BeforeAndAfterAll { self: Suite =>
   // TODO: Instead of hard-coding the path, use the findJar function from
   // Apache Spark's IntegrationTestUtils.scala.
   private val deltaConnectJar = s"$buildLocation/" +
-    "spark-connect/server/target/scala-2.13/delta-connect-server-assembly-3.3.0-SNAPSHOT.jar"
-
+    "spark-connect/server/target/scala-2.13/delta-connect-server-assembly-3.4.0-SNAPSHOT.jar"
   private val resources = s"$buildLocation/spark-connect/client/target/scala-2.13/resource_managed/test"
-  private val sparkConnectJar = s"$resources/spark-connect.jar"
-  private val sparkSubmit = s"$resources/spark/spark-4.0.0-preview1-bin-hadoop3/sbin/start-connect-server.sh"
+  private val sparkSubmit = s"$resources/spark/spark-4.0.0-preview1-bin-hadoop3/sbin/start-connect-server.sh" // HUH
```

Comment:
Should be updated to `/spark/spark-4.0.0-bin-hadoop3/sbin/start-connect-server.sh` right?

Reply:
Yeah, my bad.

It seems like we want to use `io.delta.tables.SimpleDeltaConnectService`, in that case we should be using `bin/spark-submit` instead. The `sbin/start-connect-server.sh` already uses spark-submit with a different runnner (`org.apache.spark.sql.connect.service.SparkConnectServer`). The other potential problem is that the `sbin/start-connect-server.sh` runs in daemon mode, so killing the process that started it will not kill the actual server.

Reply:
Anyway. I will undo this comment. We will pick this up in the next PR.